tests/test_modeling_common.py

```diff
@@ -1921,7 +1924,11 @@ def test_resize_tokens_embeddings(self):
 
             # Test when `vocab_size` is smaller than `hidden_size`.
             del model
+            del config
+            # Copy again. config changed with embedding resizing (`vocab_size` changed)
+            config = copy.deepcopy(original_config)
             config.vocab_size = 4
+            config.pad_token_id = 3
```

Comment:
what was this failing?

Reply:
The pad_toke_id in the config is 98 for the GitModel. This results in an error in the embedding layer because it's higher than the vocab_size
This error appeared in the GitModel after fixing the error which was caused by overwriting the configuration.